app-lifelike/src/main/java/com/example/lifelike/composable/loggedin/PhotosOfAlbum.kt

```diff
@@ -84,8 +79,43 @@ interface PhotosOfAlbum {
             val image = imageResource(R.drawable.placeholder)
 
 
+            val photoRows =  mutableListOf<List<Photo>>()
+
+            for (i in album.photos.indices step cols) {
+                val row = mutableListOf<Photo>()
+                for (r in 0 until cols){
+                    if (i+r >= album.photos.size) break
+                    row.add(album.photos[i+r])
+                }
+                photoRows.add(row)
+            }
+
             Box(modifier = Modifier.padding(4.dp)) {
-                Table(columns = cols, columnWidth = { TableColumnWidth.Fraction(1.0f / cols) }) {
+                //scrolling fast may cause exception: https://issuetracker.google.com/issues/154653504
+                AdapterList(
```

Comment:
I think the way of writing adapter list without parameters is a bit more expressive
E.g.
```
AdapterList(photoRows) { row ->
     ...
}
```